pkg/gen/awaitComments.go

```diff
@@ -15,15 +15,46 @@
 package gen
 
 import (
+	"fmt"
+
 	"github.com/pulumi/pulumi-kubernetes/pkg/kinds"
 )
 
-const preamble = `This resource waits until it is ready before registering success for
+func timeoutComment(kind kinds.Kind) string {
+	const timeoutOverride = `setting 'pulumi.com/timeoutSeconds' as a '.metadata.annotation' on the resource.`
```

Comment:
Note that this will be updated to use `customTimeouts` as soon as it's implemented.

Reply:
Is it worth linking to https://github.com/pulumi/pulumi-kubernetes/issues/672 for readers?

Reply:
Sure, might as well.